src/scripts/app.js

```diff
@@ -37,12 +49,13 @@ class App extends JSXComponent {
         return response.json();
       })
       .then((response) => {
-        if (response.statusCode >= 400) {
+        
+        // Response doesn't contain status code in here.
+        if (!response.hasOwnProperty("html") ) {
```

Comment:
That is a suuuuper edge case. I don't think we should be doing this, `statusCode` makes more sense.